database/user.go

```diff
@@ -54,19 +56,20 @@ type User struct {
 	db  *Database
 	log log.Logger
 
-	MXID           id.UserID
-	DiscordID      string
-	DiscordToken   string
-	ManagementRoom id.RoomID
-	SpaceRoom      id.RoomID
-	DMSpaceRoom    id.RoomID
+	MXID             id.UserID
+	DiscordID        string
+	DiscordToken     string
+	ManagementRoom   id.RoomID
+	SpaceRoom        id.RoomID
+	DMSpaceRoom      id.RoomID
+	HeartbeatSession *discordgo.HeartbeatSession
```

Comment:
not sure whether it's best to keep this as a pointer or not, but adding an `IsZero` method to `HeartbeatSession` felt more "dirty" compared to an explicit nil check